### ISSUE TITLE
Is 'sometimes' accurate?

### ISSUE THREAD

**Issue by lwolberg** [w3c/wcag#2305]
Paragraph 5.4 leads with, "[Sometimes, Web pages are created that will later have additional content added to them.](https://www.w3.org/TR/WCAG22/#conformance-partial)" Are you sure you want to keep "Sometimes" in this sentence? 

In addition, this AGWG note draft states, "Large websites often have complex content publishing pipelines, which may render content dynamically depending upon a large number of variables.... " [2]

[1] https://almanac.httparchive.org/en/2021/third-parties
[2] https://www.w3.org/TR/accessibility-conformance-challenges/#Challenge-2

**Comment by alastc** [comment]
> Are you sure you want to keep "Sometimes" in this sentence?

Sometimes web pages are created which do not have content added later, so I think the "sometimes" is accurate.

**Comment by JaninaSajka** [comment]
It's an adverb indicating how often. Webster's Dectionary provides these synonyms: "At times; at intervals; not always; now and then;
        occasionally." Given that the Web Almanac data cited by the commentor indicates the average web site in 2021 uses over 20 third parties, and 10% of sites use over 90 third party injections, it stretches credulity to stick with "sometimes," imo. If, as I suspect, this can't be managed in current time frames, a more honest adverb might prove far more credible. How about something like "increasingly?"

**Comment by mbgower** [comment]
I couldn't find an occurrence of "sometimes" in the second link provided. Without it, I don't have much context, but this isn't normative language, it's informative right?
If so, given the much larger challenges being tackled by 3.0, opening an issue to word-smith a single word in a draft document seems a bit counter productive.
A PR with a range of copy editing tweaks that improve but do not substantively change the meaning is probably the best way to get this level of change tackled, since it can be quickly reviewed and adopted if the editors think it is uncontroversial.

**Comment by lwolberg** [comment]
The main link to Web Content Accessibility Guidelines (WCAG) 2.2 paragraph 5.4 was left out above. I added it above and put it here: 

"[Sometimes, Web pages are created that will later have additional content added to them.](https://www.w3.org/TR/WCAG22/#conformance-partial)" 
Link. https://www.w3.org/TR/WCAG22/#conformance-partial

**Comment by mbgower** [comment]
I don't think it hurts to make this "often" but I also don't see that if really affects the meaning or context of this section either way. I suggest you submit a PR to update any similar minor editorial suggestions which don't intrinsically change the meaning.

**Comment by lwolberg** [comment]
We see the same paragraph was carried forward to WCAG 2.1
https://www.w3.org/TR/WCAG21/#conformance-partial

There are to my knowledge only a **vanishingly small number of sites** that lack the features listed in the paragraph. And this list is not even complete, skipping over e-commerce catalogs that add products and product features (e.g. sizes). 

`...a blog, an article that allows users to add comments, or applications supporting user-contributed content.... content aggregated from multiple contributors, or sites that automatically insert content from other sources over time, such as when advertisements are inserted dynamically.`

"Often" is better, @mbgower. "In general" would be more accurate. 

IMHO it's not a minor suggestion, as it points to the reality of most websites today.

**Comment by mbgower** [comment]
I think "in general" may be stretching it. There are a LOT of stable web pages out there (think of all the documentation and information pages). I like @JaninaSajka suggestion of "increasingly."

**Comment by alastc** [comment]
> Given that the Web Almanac data cited by the commentor indicates the average web site in 2021 uses over 20 third parties, and 10% of sites use over 90 third party injections, it stretches credulity to stick with "sometimes," imo. 

There is a big difference between "injections" and things which WCAG would be concerned with.

For example, tracking scripts don't (or shouldn't) change the interface at all. Images delivered by a CDN may be from a 3rd party site, but the 1st party site should be defining the alt-text. A JavaScript library might be included via a 3rd party, but the components built with it are probably  the responsibility of the 1st party.

This section is more concerned with things like user-comments, aggregated content, etc. Things which affect the interface and are from a 3rd party.

Thinking about my companies clients (mostly national & a few international level), few have needed (or wanted) to make use of partial conformance.

Some sites which accept user-comments could restrict what formatting/media is available and not need to claim partial conformance. Some may have a contract for the 3rd party that requires accessibility.

Obviously platforms like github, twitter, & other social media companies are different, but if you are looking at a site by site basis, I don't think it's obvious that it is 'most'. It might be, but it's hard to say.

I also don't think an adjective which implies change is suitable here. We don't know if it is increasing, has peaked, or how it might change in future.

Overall I don't think "some" is a problem. If that implies "a minority" to some people then perhaps we could use "many".

Or just avoid the adjective and have something like: "Web pages that will later have additional content added to them may wish to use a 'statement of partial conformance'".

**Comment by JaninaSajka** [comment]
[Accessible Platform Architectures (APA) Working Group](https://www.w3.org/WAI/APA/) has reached a [formal consensus regarding this issue during its W3C mandated horizontal review of the WCAG 2.2 Candidate Recommendation draft](https://lists.w3.org/Archives/Public/public-apa-admin/2022Aug/0059.html). Our decision accepts the second option above proposed by @alastc. We agree that dropping any adverbial or adjectival characterization is to be preferred.

**Comment by michael-n-cooper** [comment]
Reopening until a PR is processed that addresses this.

**Comment by michael-n-cooper** [comment]
Also adding the WCAG 2.2 label, as this is a horizontal review comment blocking 2.2 even though it is on 2.0 content.

**Comment by alastc** [comment]
Closing as #2658 updated this.
